openapi.yml

```diff
@@ -46,6 +46,44 @@ components:
       name: api_key
       in: query
   schemas:
+    Holding:
+      description: location holdings information
+      type: object
+      properties:
+        location:
+          type: string
+        call_number:
+          type: string
+        status:
+          type: string
+    Identifiers:
+      description: standardized identifiers
+      type: object
+      properties:
+        isbns:
+          type: array
+          items:
+            type: string
+        issns:
+          type: array
+          items:
+            type: string
+        doi:
+          type: string
+    Link:
+      description: a url related to this item
+      type: object
+      properties:
+        text:
+          type: string
+        url:
+          type: string
+          format: uri
+        restrictions:
+          type: string
+        kind:
+          description: what kind of link this is, such as full_text, restricted_full_text, sfx, unknown
+          type: string
     Record:
       description: An abbreviated representation of a record
```

Comment:
What determines which fields make it into the abbreviated record? Just wondering if that's something we want to discuss further.

Reply:
tldr; Bento uses what is in the abbreviated record in the initial search results and the full record for the, er, full record view.

We could eliminate that distinction and always return full records in the search results view as well as the full record view (we still need both API end points, but they could return the same models if we aren't concerned with performance which honestly now that you called this out I suspect would be negligible).